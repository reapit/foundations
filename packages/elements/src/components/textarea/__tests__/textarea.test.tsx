import React from 'react'
import { shallow } from 'enzyme'
import { TextArea } from '..'

describe('TextArea component', () => {
  it('should match a snapshot', () => {
    const wrapper = shallow(<TextArea />)
    expect(wrapper).toMatchSnapshot()
  })
})
