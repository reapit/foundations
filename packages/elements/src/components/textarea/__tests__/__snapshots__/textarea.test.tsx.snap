// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea component should match a snapshot 1`] = `<ElTextArea />`;
